--- aiccu/doc/aiccu.init.rpm.orig	2007-09-21 21:15:04.000000000 -0500
+++ aiccu/doc/aiccu.init.rpm	2007-09-21 21:19:19.000000000 -0500
@@ -8,6 +8,20 @@
 # config:  /etc/aiccu.conf
 # chkconfig: - 59 73
 # processname: aiccu
+#
+### BEGIN INIT INFO
+# Provides: aiccu
+# Required-Start: $local_fs $remote_fs $network $time $named
+# Required-Stop: $local_fs $remote_fs $network $time $named
+# Default-Start:
+# Default-Stop:
+# Short-Description: hearbeat daemon for IPv6-in-IPv4 (Proto-41, AYIYA, Heartbeat) tunnels
+# Description:  aiccu automatically gives one IPv6 connectivity
+#   without having to manually configure interfaces etc.
+#   One does need a SixXS account and at least a tunnel. These
+#   can be freely & gratis requested from the SixXS website.
+#   For more information about SixXS check http://www.sixxs.net
+### END INIT INFO
 
 # Source function library.
 if [ -f /etc/init.d/functions ] ; then
@@ -90,6 +104,9 @@
   reload)
   	restart
 	;;
+  force-reload)
+	restart
+	;;
   status)
   	rhstatus
 	;;
