--- doc/aiccu.init.rpm.orig	2006-04-22 16:09:38.000000000 -0500
+++ doc/aiccu.init.rpm	2006-04-22 16:10:07.000000000 -0500
@@ -87,6 +87,9 @@
   restart)
   	restart
 	;;
+  reload)
+        restart
+	;;
   status)
   	rhstatus
 	;;
@@ -94,7 +97,7 @@
   	[ -f /var/lock/subsys/aiccu ] && restart || :
 	;;
   *)
-	echo $"Usage: $0 {start|stop|restart|status|condrestart}"
+	echo $"Usage: $0 {start|stop|restart|reload|status|condrestart}"
 	exit 1
 esac
 
